src/Microsoft.AspNetCore.Http/Features/FormOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.WebUtilities;
+
+namespace Microsoft.AspNetCore.Http.Features
+{
+    public class FormOptions
+    {
+        public bool BufferBody { get; set; } = false;
+        public int MemoryBufferThreshold { get; set; } = 1024 * 64;
+        public long BufferBodyLengthLimit { get; set; } = 1024 * 1024 * 128;
+        public int KeyCountLimit { get; set; } = FormReader.DefaultKeyCountLimit;
+        public int KeyLengthLimit { get; set; } = FormReader.DefaultKeyLengthLimit;
+        public int ValueLengthLimit { get; set; } = FormReader.DefaultValueLengthLimit;
+        public int MultipartBoundaryLengthLimit { get; set; } = 128;
+        public int MultipartHeadersCountLimit { get; set; } = MultipartReader.DefaultHeadersCountLimit;
+        public int MultipartHeadersLengthLimit { get; set; } = MultipartReader.DefaultHeadersLengthLimit;
+        public long MultipartBodyLengthLimit { get; set; } = 1024 * 1024 * 128;
+    }
```

Comment:
The constants are the defaults available on lower level components. The options with magic numbers are implemented directly at the FormFeature level. Would you prefer FormFeatureConstants.DefaultMultipartBodyLengthLimit ? Or FormFeature.DefaultMultipartBodyLengthLimit?